core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.html

```diff
@@ -1,4 +1,25 @@
 <div class="oppia-contributions-show-review">
+  <div class="oppia-contributions-dropdown-container">
+    <label for="oppia-contributions-tasklist-dropdown">View: </label>
```

Comment:
In the mocks, there's no colon after "View": https://xd.adobe.com/view/f1cef5ab-0e29-4c93-ba85-7600786b57a1-a49d/. Should we stick to that? The mocks also seem to be less bold.